alfred-api-rest/src/main/resources/alfresco/module/alfred-api/module-context.xml

```diff
@@ -1,7 +1,13 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <beans xmlns="http://www.springframework.org/schema/beans"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://www.springframework.org/schema/beans
-        http://www.springframework.org/schema/beans/spring-beans-3.0.xsd">
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+            http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+            http://www.springframework.org/schema/context
+            https://www.springframework.org/schema/context/spring-context.xsd">
     <bean class="eu.xenit.alfred.api.rest.SpringConfig"/>
-</beans>
+
+    <context:component-scan base-package="eu.xenit.alfred.api.rest.v1.nodes"/>
```

Comment:
Why was this added exactly? I don't see any change in the code structure that warrants this, but perhaps I'm missing something.